staker-rewards/rewards.js

```diff
@@ -77,15 +78,31 @@ function validateIntervalTimestamps(interval) {
 
 async function calculateOperatorsRewards(context, interval) {
   const { cache } = context
+
   const slaCalculator = await SLACalculator.initialize(context, interval)
+  const fraudDetector = await FraudDetector.initialize(context)
 
   const operatorsRewards = []
 
   for (const operator of getOperators(cache)) {
+    const isFraudulent = await fraudDetector.isOperatorKeepECDSAFraudulent(
+      operator
+    )
+
+    if (isFraudulent) {
+      console.log(
+        clc.red(`operator [${operator}] has been fraudulent in Keep ECDSA`)
+      )
```

Comment:
Do we need this log since we're passing this flag to the output table?

Reply:
I wanted to bring special attention to this operator. Now it's redundant due to the changes we made. I'll remove it.